apps/token-manager/contracts/TokenManager.sol

```diff
@@ -62,6 +62,11 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
     event NewVesting(address indexed receiver, uint256 vestingId, uint256 amount);
     event RevokeVesting(address indexed receiver, uint256 vestingId, uint256 nonVestedAmount);
 
+    modifier onlyToken() {
+        require(msg.sender == address(token), ERROR_ON_TRANSFER_WRONG_SENDER);
```

Comment:
I guess the `ON_TRANSFER` part of the name should be changed now, plus now `ERROR_PROXY_PAYMENT_WRONG_SENDER` is not used anymore.